.github/workflows/test.yml

```diff
@@ -214,6 +214,7 @@ jobs:
         env:
             ETHEREUM_URL: ${{ secrets.ETHEREUM_URL }}
             AZLE_USE_DOCKERFILE: ${{ matrix.azle_source == 'repo' }}
+            AZLE_PLAINTEXT_IDENTITY: true
```

Comment:
Should we not let them pass in the storage mode that they want?

Reply:
sure

Reply:
I'm a little concerned about the quotes now